cmake/VcpkgInstallDeps.cmake

```diff
@@ -19,12 +22,67 @@ if(NOT EXISTS "${PROJ_DATA_PATH}/proj.db")
     message(FATAL_ERROR "proj.db not found at ${PROJ_DATA_PATH}/proj.db")
 endif()
 
-install(DIRECTORY "${PROJ_DATA_PATH}/" DESTINATION "${CMAKE_INSTALL_DATADIR}/proj")
-install(DIRECTORY "${VCPKG_BASE_DIR}/share/gdal/" DESTINATION "${CMAKE_INSTALL_DATADIR}/gdal")
-install(DIRECTORY "${VCPKG_BASE_DIR}/bin/Qca/" DESTINATION "bin/Qca") # QCA plugins
-install(DIRECTORY "${VCPKG_BASE_DIR}/Qt6/" DESTINATION "bin/Qt6") # qt plugins (qml and others)
+install(DIRECTORY "${PROJ_DATA_PATH}/" DESTINATION "${QGIS_DATA_SUBDIR}/proj")
+install(DIRECTORY "${VCPKG_BASE_DIR}/share/gdal/" DESTINATION "${QGIS_DATA_SUBDIR}/gdal")
+if(MSVC)
+  install(DIRECTORY "${VCPKG_BASE_DIR}/bin/Qca/" DESTINATION "${QGIS_LIB_SUBDIR}/Qca") # QCA plugins
```

Comment:
```suggestion
  install(DIRECTORY "${VCPKG_BASE_DIR}/bin/Qca/crypto/" DESTINATION "${QGIS_LIB_SUBDIR}/Qt6/plugins/crypto") # QCA plugins
```
?

Reply:
Likely yes.
If you have a possibility to confirm this manually (move the files) that would be great. No Windows right here ...

Reply:
Yep, copying "crypto" folder under Qt6/plugins fix the auth system. No more message at QGIS opening